Feature/roman

blackio opened the pull request.

Commit:
pretty basic structuring main page according to the revision d793fe5

Commit:
Add Roman Vikulov to Contributors

Commit:
add Courses Page (nav menu only)
*.html + js for mobile menu
BootStrap

Commit:
add * Black Icon Bar For Mobile MENU
add * Vertical Style For Mobile MENU ELEMENTS

Commit:
add Logo

Commit:
add Content 2*columns
add Courses 3*columns 2*line

Commit:
add Footer
add Style for Nav Menu
add text color for Body

alexbaumgertner commented on html-prototype/courses.html:
```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+	<meta charset="utf-8">
```
Нужно поправить отступы, перевести `Tab` в 4 пробела.

Commit:
add bootstrap.js as link

Commit:
fixed Tabs and removed attribute "role" for <ul>

Commit:
fixed Footer

Commit:
add Links for <h1> and <h3> add  Links style in css

Commit:
add Semantic tags: <header> <section>
deleted some <div> tags and changed style for header and margin

Commit:
Add Roman Vikulov to Contributors

Commit:
deleted 2 columns content element below nav menu
add Semantic tags : <section> <figure> <figcaption>
add <h3> description for course Type
css: deleted some unnecessary stuff

alexbaumgertner commented:
:ping:

alexbaumgertner commented:
@vikulov-roman поправь пожалуйста отступы.

Commit:
add Join Us block, fix Tabs

Commit:
Merge branch 'feature/roman' of https://github.com/freecodecamp-spb/web-dev-cources into feature/roman

# Conflicts:
#	css-prototype/main.css

Commit:
discard styles for divs
